File: Trunk/vpPriV100GrupoMundifios/Default/Plataforma/PltNsEmpresas.cs
using Generico;
using Primavera.Extensibility.BusinessEntities.ExtensibilityService.EventArgs;
using Primavera.Extensibility.Platform.Services;
using StdBE100;

namespace Default
{
    public class PltNsEmpresas : Plataforma
    {
        // *******************************************************************************************************************************************
        // #### ARMAZEM ENTREPOSTO ####
        // *******************************************************************************************************************************************
        private StdBELista ListaArmEnt;

        private string SqlStringArmEnt;
        // *******************************************************************************************************************************************
        // #### ARMAZEM ENTREPOSTO ####
        // *******************************************************************************************************************************************

        public override void DepoisDeAbrirEmpresa(ExtensibilityEventArgs e)
        {
            base.DepoisDeAbrirEmpresa(e);

            if (Module1.VerificaToken("Default") == 1)
            {
                // *******************************************************************************************************************************************
                // #### ARMAZEM ENTREPOSTO ####
                // *******************************************************************************************************************************************
                SqlStringArmEnt = "SELECT CDU_Parametro FROM TDU_Parametros WHERE CDU_Modulo = 'Entreposto'";

                ListaArmEnt = BSO.Consulta(SqlStringArmEnt);

                if (ListaArmEnt.Vazia() == false)
                    Module1.ArmEntreposto = ListaArmEnt.Valor("CDU_Parametro");
            }
        }
    }
}

File: Branches/VBInicial/vpPriV100GrupoMundifios/Default/Plataforma/PltNsEmpresas.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports Primavera.Extensibility.Platform.Services
Imports Primavera.Extensibility.BusinessEntities
Imports Primavera.Extensibility.BusinessEntities.ExtensibilityService.EventArgs
Imports StdBE100
Imports Vimaponto.PrimaveraV100.Clientes.GrupoMundifios.Generico

Namespace DefaultToken
    Public Class PltNsEmpresas
        Inherits Plataforma



        '*******************************************************************************************************************************************
        '#### ARMAZEM ENTREPOSTO ####
        '*******************************************************************************************************************************************
        Dim ListaArmEnt As StdBELista
        Dim SqlStringArmEnt As String
        '*******************************************************************************************************************************************
        '#### ARMAZEM ENTREPOSTO ####
        '*******************************************************************************************************************************************

        Public Overrides Sub DepoisDeAbrirEmpresa(e As ExtensibilityEventArgs)
            MyBase.DepoisDeAbrirEmpresa(e)

            If Module1.VerificaToken("Default") = 1 Then
                '*******************************************************************************************************************************************
                '#### ARMAZEM ENTREPOSTO ####
                '*******************************************************************************************************************************************
                SqlStringArmEnt = "SELECT CDU_Parametro FROM TDU_Parametros WHERE CDU_Modulo = 'Entreposto'"

                ListaArmEnt = BSO.Consulta(SqlStringArmEnt)

                If ListaArmEnt.Vazia = False Then
                    Module1.ArmEntreposto = ListaArmEnt.Valor("CDU_Parametro")
                End If
                '*******************************************************************************************************************************************
                '#### ARMAZEM ENTREPOSTO ####
                '*******************************************************************************************************************************************

            End If

        End Sub

    End Class
End Namespace
